README.md

```diff
@@ -15,29 +15,29 @@ as **T-F-D**:
 ## Types
 
  | Type code   | Referencing        |
- | ----------- | ------------------ |
- | 0           | Peer ID            |
- | 1           | Message ID         |
- | 2           | Blob ID            |
- | 3           | Diffie-Hellman key |
- | 4           | Signature          |
- | 5           | Encrypted data     |
- | 6           | Generic data       |
-
-### Peer ID formats
-
-A peer ID TFD represents the public portion of a cryptographic keypair used to
+ |:-----------:| ------------------ |
+ | 0           | feed               |
+ | 1           | msg                |
+ | 2           | blob               |
+ | 3           | key                |
+ | 4           | signature          |
+ | 5           | encrypted          |
+ | 6           | generic            |
+
+### 0. feed formats
```

Comment:
Same here, as a specification I would expect we are writing for humans in a formal manner, and we use uppercase.

Reply:
Agree with this one. I think the table should have programmatic content and the title should be more human.
Still like the number code there to really hammer out home

Reply:
Capital cased the titles